src/main/steps/common/hmcts-progress-bar/_progress-bar.scss

```
@@ -0,0 +1,127 @@
+/* ==========================================================================
+   #PROGRESS BAR
+   ========================================================================== */
+
+.hmcts-progress-bar {
+  margin-bottom: govuk-spacing(7);
+}
+
+.hmcts-progress-bar__list {
+  font-size: 0; // Hide white space between elements
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  position: relative;
+  text-align: justify;
+  vertical-align: top;
+
+  // IE8 does not support the text justify approach for spacing
+  @include govuk-if-ie8 {
+    display: table;
+    table-layout: fixed;
+    width: 100%;
+  }
+
+  &::after {
+    content: "";
+    display: inline-block;
+    width: 100%;
+  }
+
+  &::before {
+    border-top: 6px solid govuk-colour("green");
+    content: "";
+    left: 0;
+    position: absolute;
+    top: 13px;
+    width: 100%;
+  }
+
+}
+
+
+.hmcts-progress-bar__list-item {
+  @include govuk-font(19);
+  display: inline-block;
+  max-width: 20%;
+  position: relative;
+  text-align: center;
+  vertical-align: top;
+
+  // IE8 does not support the text justify approach for spacing
+  @include govuk-if-ie8 {
+    display: table-cell;
+  }
+
+  &:first-child,
+  &:last-child {
+    &::before {
+      border-top: 6px solid govuk-colour("white");
+      content: "";
+      position: absolute;
+      top: 13px; left: 0;
+      width: 50%;
+    }
+
+  }
+
+  &:first-child {
+
+    &::before {
+      left: 0;
+    }
+
+  }
+
+  &:last-child {
+
+    &::before {
+      left: auto;
+      right: 0;
+    }
+
+  }
+
+  &[aria-current=step] { // https://tink.uk/using-the-aria-current-attribute
+    font-weight: 700;
+  }
+
+}
+
+
+.hmcts-progress-bar__icon {
+  position: relative;
+  background-color: govuk-colour("white");
+  border: 6px solid govuk-colour("green");
+  border-radius: 50%; // IE8 does not support border radius but we’re ok with that
+  box-sizing: border-box;
+  display: block;
+  height: 32px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 32px;
+}
```

Should we move this file into the scss folder and put in the progress bar styles from main.scss? After that we will have all our progress bar styles in one place, e.g. "assets/scss/progress-bar.scss"

We could combine the css where we use the same classes in both files. For example, we currently override all the styles of this class in main.scss, which means it can become:
```suggestion
.hmcts-progress-bar__icon {
  position: relative;
  background-color: govuk-colour('white');
  border: 4px solid govuk-colour('mid-grey');
  border-radius: 50%;
  box-sizing: border-box;
  display: block;
  height: 44px;
  margin-left: auto;
  margin-right: auto;
  width: 44px;
}
```